esp/services/ws_smc/ws_smcService.cpp

```diff
@@ -1407,7 +1402,7 @@ bool CWsSMCEx::onMoveJobDown(IEspContext &context, IEspSMCJobRequest &req, IEspS
             }
         }
         AccessSuccess(context, "Changed job priority %s",req.getWuid());
-        activityInfoReader->rebuild();
+        activityInfoCacheReaderThread->rebuild();
```

Comment:
This is just a name change. The activityInfoReader is renamed as activityInfoCacheReaderThread because CActivityInfoReader is renamed as CInfoCacheReaderThread (please see PR comment 1 and 2).